src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -14180,6 +14356,41 @@ Label900: ;
 		return StringOut;
 	}
 
+	bool 
+	isNumber(std::string s)
+	{
+		bool is = true;
+		try {
+			std::stof(s);
+		} catch ( std::exception & e ) {
```

Comment:
@Myoldmopar I know you suggested this, but I disagree with using a try/catch to check if it is a number. That is an expensive paradigm that I don't want to spread in E+ code. Also, I don't think `stof()` will function properly for very large numbers since you are checking the broad `std::exception`. `stof` throws `std::invalid_argument` when out of range of floats (which could happen since we are using doubles in all of E+). You should use `std::stod`.

An alternate approach is to use `strtod`:
```cpp
bool
isNumber(std::string const & s) {
  char* p;
  strtod( s.c_str(), &p, 10 );
  return *p == 0;
}
```